issue1284670.patch

```diff
@@ -0,0 +1,171 @@
+--- a/Lib/modulefinder.py
```

Comment:
the `issue1284670.patch`  file should not be part of the PR, and I'm guessing that `_bootstrap_python` should also not be part of the PR

Reply:
I have tried to delete in my Github copy but I can only delete _bootstrap_python. Github doesn't allow me to delete the patch file in my cpython copy. How do I get rid of it?